apps/em/em-hrs-ingestor/perftest-image-policy.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: image.toolkit.fluxcd.io/v1beta1
+kind: ImagePolicy
+metadata:
+  name: perftest-em-hrs-ingestor
+  annotations:
+    hmcts.github.com/prod-automated: enabled
+spec:
+  filterTags:
+    pattern: '^prod-[a-f0-9]+-(?P<ts>[0-9]+)'
```

Comment:
Are you changing this from default prod?

Reply:
No, plan is to keep it the same but have the setup with image policies for all the lower envs to be able to deploy other things in. For now it is just to have them pulling the latest prod image. Perftest currently is pulling some hardcoded image from 2021 (?) and is not able to create pods.